exercises/currying/Curry1.hs

```diff
@@ -0,0 +1,49 @@
+module Curry1 where
+
+-- I AM NOT DONE
+
+{-
+- In Haskell, functions that take more than one argument are usually
+  curried, meaning the function takes one argument and returns another function that takes the second argument. 
+  
+- The standard library defines a *higher order function* *curry* which 
+  takes a function from a pair (a,b) -> c as an argument and returns a function that takes a and returns a function b -> c.
+  
+curry :: ((a, b) -> c) -> a -> b -> c
+
+- The inverse of *curry* is called *uncurry*.
+
+uncurry :: (a -> b -> c) -> (a, b) -> c
+
+-}
+
+-- The manhatten distance of two coordinat points is defined as the distance in x + the distance in y. 
+-- This function takes the xdistance and ydistance and calculates the manhatten distance:
+
+
+
+manhattenDistance :: Int -> Int -> Int
+manhattenDistance xdist ydist = abs xdist + abs ydist
+
+-- TODO:
+
+-- Define a *type synonym* 'Point' for a pair (Int, Int).
```

Comment:
In terms of organization, I actually like the idea of not using a type synonym here so that we can place it within the `functions` directory.

We can just say "Let's imagine that a pair, `(Int, Int)` represents a 2D point.